crates/cxx-qt-gen/src/generator/naming/property.rs

```diff
@@ -12,22 +12,49 @@ pub struct QPropertyNames {
     pub name: Name,
     pub getter: Name,
     pub getter_wrapper: Name,
-    pub setter: Name,
-    pub setter_wrapper: Name,
-    pub notify: Name,
+    pub setter: Option<Name>,
```

Comment:
this would then become the enum QPropertyFlagState.

There would be the question over whether the getter ones should have an enum as well to do the same but without None as an option?

Reply:
done